core/capabilities/launcher.go

```
@@ -295,6 +296,12 @@ func (w *launcher) addRemoteCapabilities(ctx context.Context, myDON registrysync
 			return fmt.Errorf("could not unmarshal capability config for id %s", cid)
 		}
 
+		methodConfig := capabilityConfig.CapabilityMethodConfig
+		if methodConfig != nil { // v2 capability - handle via CombinedClient
+			w.addRemoteCapabilityV2(ctx, capability.ID, methodConfig, myDON, remoteDON, state)
```

you are missing the potential error when adding the capability, if not will fail silently, right?